### PR TITLE
add wait_for_status instead of wait_for_active

### PR DESCRIPTION
**Pull request by sed-i**

### CODE REVIEW

**Review by johnsca — COMMENTED** [review]
Looks reasonable to me. However, there are some integration tests which use `wait_for_idle(wait_for_active=True)` and if that is going to be deprecated, those should be updated (and maybe a regression test included).

**Comment by sed-i** [comment]
Updated integration tests and added unit tests.

**Comment by sed-i** [comment]
@johnsca how about renaming `wait_for_status` to simply `status`, such that `await ops_test.model.wait_for_idle(status="active")`

**Comment by johnsca** [comment]
+1 to `status`.  That seems plenty clear and much less typing.

**Comment by sed-i** [comment]
@johnsca: renamed to status and patched timedelta

**Review by johnsca — APPROVED** [review]

File: tox.ini [hunk 1]
```diff
@@ -10,6 +10,9 @@ skipsdist=True
 [pytest]
 markers =
     serial: mark a test that must run by itself
+filterwarnings =
+    ignore::DeprecationWarning:asynctest
+    ignore::DeprecationWarning:websockets
```
Comment (SimonRichardson) [inline tox.ini:15]: :clap:

**Comment by SimonRichardson** [comment]
!!build!!

**Review by SimonRichardson — APPROVED** [review]

**Comment by SimonRichardson** [comment]
$$merge$$

**Comment by SimonRichardson** [comment]
$$merge$$

**Pull request merged**